### ISSUE TITLE
preprocessing script?

### ISSUE THREAD

**Issue by djamahl99** [KTH-RPL/SeFlow#2]
I hope this message finds you well. I am currently trying to rerun your code but noticed that there is no implementation for 0_preprocess.py. The script indicates that I should run dufomap, but I am unsure about the specifics of how the dataset was created for training. Could you kindly provide this script or offer some guidance on how to proceed?

Thank you very much for your assistance.

**Comment by Kin-Zhang** [comment]
We will open it soon at the same time with paper available in Springer. I will remind here when it’s public.
Btw I directly upload a mini demo proceeded dataset and also pretrain weight of SeFlow.

**Comment by djamahl99** [comment]
Hi, I can see you have given a link to download the preprocessing on Waymo but AV2 does not seem to have a link?

E.g. waymo you gave:
```
wget https://zenodo.org/records/12751363/files/waymo_map.tar.gz
```

Below you have given a hyperlink to preprocessing which is broken (preprocess folder does not exist). 
![image](https://github.com/user-attachments/assets/be779923-b604-431b-9873-4b366a87bb4f)

**Comment by Kin-Zhang** [comment]
Hi, I think you misunderstood the demo data, the one you attached it's the preprocessing for Waymo, not Waymo demo data. The demo data link is: https://zenodo.org/records/12751363/files/demo_data.zip?download=1 for av2

I didn't upload any waymo demo data since their license constraint.

**Comment by djamahl99** [comment]
Hi, thanks for helping me. I was confused as I thought you were saying you had only omitted preprocessing for nuScenes but indeed you have not provided preprocessing for any datasets (to get labels for SSL). It is necessary that I retrain with your method for my paper so I will have to wait for your preprocessing technique to be released.  I am eager to trial your method in future :).

Thanks,
Djamahl

**Comment by Kin-Zhang** [comment]
> We will open it soon at the same time as the paper available in Springer. I will remind you here when it’s public.
> 
> Guidance: It uses [dufomap](https://github.com/KTH-RPL/dufomap/tree/main) and [HDBSCAN](https://hdbscan.readthedocs.io/en/latest/index.html) as said in the paper, I attached both code links for you to have a quick look. We save the label in the h5py file which you can [follow this instruction](https://github.com/KTH-RPL/SeFlow/blob/main/dataprocess/README.md#process)
> 
> Last but not least, for people who urgently need to reproduce results, I directly uploaded the pre-trained weight of SeFlow and a mini demo proceeded dataset for a mini-training try.

I think I explained here already.

I will remind here when paper is published officially and process SSL label script will also be pushed.
